api/src/test/resources/testAppDataDir/configuration/jsonkeyvalues/config.json

```diff
@@ -26,5 +26,13 @@
     "report.newEpisodesOfDiseases.questions.conceptSet" : "8dcb0dd6-b31f-41d1-b144-d49a2c5340ab",
     "report.newEpisodesOfDiseases.allDiagnoses.conceptSet" : "dccb05f6-a27b-4331-989c-9a669e84e467",
     "report.newEpisodesOfDiseases.referredCodedanswer.concept" : "d7f6f425-df28-47f4-9474-ba85a462a261",
-    "report.newEpisodesOfDiseases.active" : "true"
+    "report.newEpisodesOfDiseases.active" : "true",
+    "report.vaccination.active" : "true",
+    "report.ecv.active" : "true",
+    "report.prenatalVisitType" : "35ba9aff-901c-49dc-8630-a59385480d18",
+    "report.vaccinations" : "3c25e650-ccfa-48da-b276-a7872796b113",
+    "report.vaccinationSequenceNumber" : "39741bfe-36a0-4a5b-9c64-fd4caaadca01",
```

Comment:
We should probably keep the naming convetion of:
`report.reportName.propertyName`

In that case: `report.vaccination.sequenceNumber`.

But it appears this does not point to an actual sequence number, and rather the sequence number concept.
If that's so, we should rename it as: `report.vaccination.sequenceNumberConcept`.

Reply:
Done

Reply:
Done